src/Common/src/Interop/Windows/user32/Interop.FindWindow.cs

```diff
@@ -0,0 +1,15 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
How's code coverage look with all your tests?

Reply:
Do we have a good way to measure that?

Reply:
https://github.com/dotnet/corefx/blob/master/Documentation/building/code-coverage.md

Reply:
That's slick.  I have 83.4% coverage.